### PR TITLE
Template for Pull Request

### PR DESCRIPTION
**Pull request by rcillo**
In order to remind developers about the minimum requirements for a pull
request, we are adding a template.

In case your PR does not require deployment notes, for example, just remove it. It would at least mean that you made a conscious decision not to include this data other than forgot about it.

### CODE REVIEW

**Comment by lmontrieux** [comment]
I guess this is ARUHA-940?

File: github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+## Description
+A few sentences describing the overall goals of the pull request's commits.
+
+## Todos
```
Comment (lmontrieux) [inline github/PULL_REQUEST_TEMPLATE.md]: I'm nitpicking, but I would call this `Review`

File: github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+- [ ] Tests
+- [ ] Documentation
+- [ ] CHANGELOG
+
+## Deploy Notes
```
Comment (lmontrieux) [inline github/PULL_REQUEST_TEMPLATE.md]: Maybe `Deployment Notes`?

File: github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+Notes regarding deployment the contained body of work. These should note any
```
Comment (lmontrieux) [inline github/PULL_REQUEST_TEMPLATE.md]: `these should note` -> `These should highlight` (so we don't repeat 'note' too often)

**Comment by rcillo** [comment]
@lmontrieux you're right. I added a link to this ticket in the ticket https://techjira.zalando.net/browse/ARUHA-940 and addressed your comments.

**Comment by lmontrieux** [comment]
👍

**Comment by rcillo** [comment]
👍

**Pull request merged**